Unix/build.mak

```diff
@@ -392,13 +392,20 @@ CHECKDIR=/tmp/omicheck.$(shell ./buildtool username)
 check:
 	rm -rf $(CHECKDIR)
 	( cd $(CONFIGUREDIR); $(OUTPUTDIR)/install --destdir=$(CHECKDIR) )
-	(  export LD_LIBRARY_PATH=$(CHECKDIR)/$(CONFIG_LIBDIR);  \
-           export DYLD_LIBRARY_PATH=$(CHECKDIR)/$(CONFIG_LIBDIR); \
-           $(CHECKDIR)/$(CONFIG_BINDIR)/omiserver -i -d --livetime 60 --httpport 0 --httpsport 0 --destdir=$(CHECKDIR) )
+	( LD_LIBRARY_PATH=$(CHECKDIR)/$(CONFIG_LIBDIR);  \
+	  export LD_LIBRARY_PATH;  \
+          DYLD_LIBRARY_PATH=$(LD_LIBRARY_PATH); \
+          export DYLD_LIBRARY_PATH; \
+          $(CHECKDIR)/$(CONFIG_BINDIR)/omiserver -i -d --livetime 60 --httpport 0 --httpsport 0 --destdir=$(CHECKDIR) )
```

Comment:
This is still not lining up properly, please fix. The problem is that line 396 is tab prefixed and the remainder are space prefixed. If you fix that, things will line up. You may want to use an editor that can display tab stops to make this more clear.